examples/generic/main.tf

```diff
@@ -0,0 +1,58 @@
+
+terraform {
+  required_version = ">= 0.13"
+
+  required_providers {
+    flux = {
+      source = "fluxcd/flux"
+    }
+    gitfile = {
+      source  = "igal-s/gitfile"
+      version = "1.0.0"
+    }
+  }
+}
+
+provider "flux" {}
+provider "gitfile" {}
+
+# Generate the flux manifest
+data "flux_install" "main" {
+  target_path = "cluster-init"
+}
+
+data "flux_sync" "main" {
+  target_path = "cluster-init"
+  url         = var.repository_url
+}
+
+# Git: Create the files and commit them to the repository
+resource "gitfile_checkout" "example" {
+  repo   = var.repository_url
+  branch = var.branch
+  path   = "${path.root}/git_checkouts/cluster-init"
```

Comment:
Would this not place the cloned repo inside of your terraform directory?

Reply:
Yes it will, u want it to be moved to path.module ?

Reply:
made it path.module